pycodestyle.py

```diff
@@ -1504,13 +1504,15 @@ def ambiguous_identifier(logical_line, tokens):
     Okay: lambda arg: arg * l
     Okay: lambda a=l[I:5]: None
     Okay: lambda x=a.I: None
+    Okay: if l >= 12:
     E741: except AttributeError as O:
     E741: with lock as l:
     E741: global I
     E741: nonlocal l
     E741: def foo(l):
     E741: def foo(l=12):
     E741: l = foo(l=12)
+    E741: while l := thing:
```

Comment:
you'll have to put this in the versioned test file rather than the docstring

Reply:
wow, super speedy. you're right, one sec

Reply:
I don't think you need the condition you've added -- `:=` will never appear as a token in 3.7

Reply:
100%, i was barking up the wrong tree